linux/shim/init.go

```diff
@@ -237,31 +237,30 @@ func (p *initProcess) SetExited(status int) {
 }
 
 func (p *initProcess) Delete(context context.Context) error {
-	status, err := p.Status(context)
-	if err != nil {
-		return err
-	}
-	if status != "stopped" {
```

Comment:
This change the behavior of the API by not requiring the task to be in the stopped state, any reason why the call to status was removed? Does `runc` loses all information during checkpoint?